mod_ci/controllers.py

```diff
@@ -463,23 +463,45 @@ def queue_test(db, gh_commit, commit, test_type, branch="master", pr_nr=0):
     """
     from run import log
 
-    fork = Fork.query.filter(Fork.github.like("%/CCExtractor/ccextractor.git")).first()
+    fork = Fork.query.filter(Fork.github.like(
+        "%/CCExtractor/ccextractor.git")).first()
+    fork_id = Test.query.filter(Test.fork_id == Fork.id).first()
+    githubapi = requests.get(
+        "https://api.github.com/repos/ccextractor/ccextractor/pulls/{}".format())
+    userjson = json.loads(githubapi.body)
+    user = userjson['user']['id']
+
+    record_from_db = BlockedUsers.query.filter_by(userID=user).first()
+    blocked = False
 
-    if test_type == TestType.pull_request:
-        branch = "pull_request"
+    if user == record_from_db.userID:
```

Comment:
This won't work if there's no matching user, because first will return None, and you can't get a property from that.